tests/core/posix.c

```diff
@@ -15,11 +15,15 @@
 #include "posix.h"
 #include "userdiff.h"
 
+#if LC_ALL > 0
```

Comment:
This is a bit weird. Does Windows define `LC_ALL` to `0` or doesn't it define it at all? If the latter's the case, then we should definitely use `#ifndef` instead. I'll also upload a PR that enables -Wundef in the short-term future to generate a warning for such uses

Reply:
Windows defines `LC_ALL` as `0`.